ci-operator/step-registry/redhat-appstudio/gather/redhat-appstudio-gather-commands.sh

```diff
@@ -21,8 +21,8 @@ function queue() {
 # Appstudio resources
 queue ${ARTIFACT_DIR}/applicationpromotionruns.json  oc --insecure-skip-tls-verify --request-timeout=5s get applicationpromotionruns.appstudio.redhat.com --all-namespaces -o json
 queue ${ARTIFACT_DIR}/applications_appstudio.json  oc --insecure-skip-tls-verify --request-timeout=5s get applications.appstudio.redhat.com --all-namespaces -o json
-queue ${ARTIFACT_DIR}/applicationsnapshotenvironmentbindings.json  oc --insecure-skip-tls-verify --request-timeout=5s get applicationsnapshotenvironmentbindings.appstudio.redhat.com --all-namespaces -o json
-queue ${ARTIFACT_DIR}/applicationsnapshots.json  oc --insecure-skip-tls-verify --request-timeout=5s get applicationsnapshots.appstudio.redhat.com --all-namespaces -o json
+queue ${ARTIFACT_DIR}/snapshotenvironmentbindings.json  oc --insecure-skip-tls-verify --request-timeout=5s get snapshotenvironmentbindings.appstudio.redhat.com --all-namespaces -o json
+queue ${ARTIFACT_DIR}/snapshots.json  oc --insecure-skip-tls-verify --request-timeout=5s get snapshots.appstudio.redhat.com --all-namespaces -o json
```

Comment:
I checked that some CRDs don't exist anymore, so perhaps we don't need to try to collect them
```
applicationpromotionruns.appstudio.redhat.com
releaselinks.appstudio.redhat.com
conditions.tekton.dev
```

otherwise lgtm

Reply:
promotionruns exists but not used yet

Reply:
@kasemAlem was `releaselinks` replaced with `releaseplans`? We should make sure we are capturing all the equivalent Release CRs

Reply:
Indeed , 
it was replaced long ago, 
I'll review it with those old CRs